sorc/ncep_post.fd/UPP_PHYSICS.f

```diff
@@ -1764,6 +1764,440 @@ SUBROUTINE CALCAPE2(ITYPE,DPBND,P1D,T1D,Q1D,L1D,    &
       END SUBROUTINE CALCAPE2
 !
 !-------------------------------------------------------------------------------------
+!
+!
+
+      SUBROUTINE CALCAPE1D(P_A,T_A,Q_A,ZINT_A,LPAR0,PSFC,LMASK,CAPE,CINS,    &
```

Comment:
Same as what I comment in MISCLN.f for scheme 2 and in TTBLEX.f for subroutine TTBLEX1D,
 "_After the testing in HWT SFE 2021, it seems that Scheme 1 had been working well and SPC was satisfied with it. So to make the code management and maintenance simple and easy, I suggest to set Scheme 1 as the only scheme for Effective-Layer, and remove the code related to Scheme 2 (in this MISCLN.f, and in UPP_PHYSICS.f and in TTBLEX.f). Just my suggestion. These code still can be kept._"
This subroutine CALCAPE1D is used only for scheme 2 in MISCLN.f for searching for the Effective-Layer. It could be removed for code clean if scheme 2 is not necessary.